common-lib/test/cmr/common/test/util.clj

```diff
@@ -48,15 +48,7 @@
       "The most basic case with 1"
       2 (+ 1 1)
       "A more complicated test"
-      4 (* 2 2)))
-  (testing "without comments"
```

Comment:
This one now throws an error at compile time, so no need for the test case. Same for `are3-test` below.